
config4 Multi
begin eval
  0%|                                                                                                                                                                                                              | 0/12500 [00:02<?, ?it/s]
Traceback (most recent call last):
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 105, in <module>
    confusion()
  File "/home/ericji3/ECE496/ECE496/src/eval_model.py", line 86, in confusion
    output = model(data)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/ericji3/ECE496/ECE496/src/architectures.py", line 44, in forward
    x = self.model(x)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torchvision/models/resnet.py", line 285, in forward
    return self._forward_impl(x)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torchvision/models/resnet.py", line 268, in _forward_impl
    x = self.conv1(x)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 460, in forward
    return self._conv_forward(input, self.weight, self.bias)
  File "/home/ericji3/miniconda3/envs/ECE496/lib/python3.9/site-packages/torch/nn/modules/conv.py", line 456, in _conv_forward
    return F.conv2d(input, weight, bias, self.stride,
RuntimeError: Given groups=1, weight of size [64, 4, 7, 7], expected input[16, 5, 256, 256] to have 4 channels, but got 5 channels instead